src/Tribe/Events/Attendee_List_Display.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Tribe\Tickets\Events;
+
+use Exception;
+use InvalidArgumentException;
+use Tribe__Tickets_Plus__Attendees_List;
+use WP_Post;
+
+class Attendee_List_Display {
+
+	private static $attendee_list_by_shortcode = 'tribe_tickets_attendee_list_triggered_by_shortcode';
+
+	/**
+	 * Check if given event is hiding the attendees list.
+	 *
+	 * @param int|WP_Post $post
+	 *
+	 * @throws InvalidArgumentException
+	 *
+	 * @return bool True if event is hiding the attendees list. False otherwise.
+	 */
+	public function is_event_hiding_attendee_list( $post ) {
+		// Attendees list is a Plus feature, if ET Plus is not present it will always be hidden.
+		if ( ! class_exists( 'Tribe__Tickets_Plus__Main' ) ) {
+			return true;
+		}
+
+		if ( is_numeric( $post ) ) {
+			$post = WP_Post::get_instance( $post );
+		}
+
+		if ( ! $post instanceof WP_Post ) {
+			/*
+			 * There's no way to apply a sensible default in this scenario.
+			 *
+			 * Let's throw an exception so the client calling this function deals
+			 * with it according to the >> caller << context.
+			 */
+			throw new InvalidArgumentException( '$post should be either a post ID or a WP_Post object.' );
+		}
+
+		return (bool) Tribe__Tickets_Plus__Attendees_List::is_hidden_on( $post );
+	}
+
+	/**
+	 * @filter tribe_tickets_plus_hide_attendees_list_optout 10 1
```

Comment:
:no_entry_sign: Doc comment for parameter "$should_hide" missing

<sub>hash: 2930b150c80210c9fff05fc7dc091ab0</sub>